static/app/components/onboarding/productSelection.tsx

```diff
@@ -135,6 +147,19 @@ export function ProductSelection({defaultSelectedProducts}: Props) {
           />
         </Product>
       </Products>
+      {lazyLoader && (
+        <AlternativeInstallationAlert type="info" showIcon>
+          {tct('Prefer to set up Sentry using [npm] or [yarn]? [goHere].', {
+            npm: <strong>npm</strong>,
+            yarn: <strong>yarn</strong>,
+            goHere: (
+              <Button onClick={skipLazyLoader} priority="link">
+                {t('Go here')}
+              </Button>
+            ),
```

Comment:
```suggestion
          {tct('Prefer to set up Sentry using [npm:npm] or [yarn:yarn]? [goHere:Go here].', {
            npm: <strong />,
            yarn: <strong />,
            goHere: (
              <Button onClick={skipLazyLoader} priority="link" />
            ),
```

Reply:
hm, I like the suggestion with the exception for the button... will it have the proper `aria-label` and `children`?